documentation/book/con-deploying-cluster-operator-using-operator-hub.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// assembly-cluster-operator.adoc
+
+[id='deploying-cluster-operator-using-operator-hub-{context}']
+= Deploying the Cluster Operator using OperatorHub.io
```

Comment:
I think the following title is more accurate:

**Deploying the Cluster Operator from OperatorHub.io**